flycheck.el

```diff
@@ -3877,10 +3902,10 @@ beginning of the buffer, otherwise advance from the current
 position.
 
 Intended for use with `next-error-function'."
-  (let ((pos (flycheck-next-error-pos n reset)))
-    (if pos
-        (goto-char pos)
-      (user-error "No more Flycheck errors"))))
+  (-if-let* ((pos (flycheck-next-error-pos n reset)))
+      (let ((err (get-char-property pos 'flycheck-error)))
```

Comment:
Isn't that `let` superfluous since we already have `-if-let*` above?

Reply:
Yes, good catch. Fixed in 0ff3c723.